config/autoscaling/staging-policy.json

```diff
@@ -1,22 +1,14 @@
 {
-  "instance_min_count": 1,
-  "instance_max_count": 3,
+  "instance_min_count": 2,
+  "instance_max_count": 12,
```

Comment:
Staging has identical thresholds to production, but with a lower minimum instance limit.